tests/testcases/a_basics/container_testcases.py

```diff
@@ -49,23 +61,216 @@ def test003_start_stop_container(self):
         #. Start container C1, should succeed.
         #. Check if the container has been started, should succeed
         """
-        
         self.logger.info('Create a container (C1)')
-        self.controller.container_manager.install(wait=True)
-        self.assertTrue(self.controller.container_manager.install_state, " Installtion state is False")
+        container = self.controller.container_manager(parent=self.controller, service_name=None)
+        container.install(wait=True)
+        self.containers.append(container)
+        self.assertTrue(container.install_state, " Installtion state is False")
+
         self.logger.info('Stop container C1, should succeed.')
-        self.controller.container_manager.stop()
+        container.stop()
         time.sleep(2)
         self.logger.info('Check if the container has been terminated, should succeed')
-        conts = self.controller.node.client.container.list()
-        self.assertFalse([c for c in conts.values() if c['container']['arguments']['hostname'] == self.controller.container_manager.data['hostname']])
+        conts = self.node.client.container.list()
+        self.assertFalse([c for c in conts.values() if c['container']['arguments']['hostname'] == container.data['hostname']])
 
         self.logger.info('Start container C1, should succeed.')
-        self.controller.container_manager.start()
+        container.start()
         time.sleep(2)
 
         self.log('Check if the container has been started, should succeed')
-        conts = self.controller.node.client.container.list()
-        self.assertTrue([c for c in conts.values() if c['container']['arguments']['hostname'] == self.controller.container_manager.data['hostname']])
+        conts = self.node.client.container.list()
+        self.assertTrue([c for c in conts.values() if c['container']['arguments']['hostname'] == container.data['hostname']])
+
+    @parameterized.expand(['before', 'after'])
+    @unittest.skip('https://github.com/threefoldtech/0-templates/issues/201')
+    def test004_create_container_with_zt_nic(self, add):
+        """ ZRT-ZOS-016
+        *Test case for creating container with zerotier network*
+        **Test Scenario:**
+
+        #. Create a container (C1).
+        #. Add zerotier network to container (c1).
+        #. Check that netwrok is added successfully to the container.
```

Comment:
Add check that container has been already joined the zerotier network form zerotier side